SignalServiceKit/src/Messages/OWSMessageManager.m

```diff
@@ -191,7 +191,9 @@ - (void)handleDeliveryReceipt:(OWSSignalServiceProtosEnvelope *)envelope
         TSOutgoingMessage *outgoingMessage = (TSOutgoingMessage *)interaction;
         [outgoingMessage updateWithWasDeliveredWithTransaction:transaction];
     } else {
-        OWSFail(@"%@ Unexpected message with timestamp: %llu", self.tag, envelope.timestamp);
+        // Desktop currently sends delivery receipts for "unpersisted" messages
+        // like group updates, so these errors are expected to a certain extent.
+        DDLogError(@"%@ Unexpected message with timestamp: %llu", self.tag, envelope.timestamp);
```

Comment:
Oh right - I suppose all clients do, since that's currently a server thing. Maybe DDLogInfo or even Debug.

Reply:
Done.